esrally/actor.py

```diff
@@ -86,15 +87,16 @@ def receiveMsg_DefuseBomb(self, msg, sender):
     """
 
     def guard(self, msg, sender):
+        # noinspection PyBroadException
         try:
             return f(self, msg, sender)
-        except BaseException as e:
-            msg = "Error in {}".format(actor_name)
+        except BaseException:
             # log here as the full trace might get lost.
-            logging.getLogger(__name__).exception(msg)
+            logging.getLogger(__name__).exception("Error in %s", actor_name)
             # don't forward the exception as is because the main process might not have this class available on the load path
             # and will fail then while deserializing the cause.
-            self.send(sender, BenchmarkFailure("{} ({})".format(msg, str(e))))
+            parsed_exception = traceback.format_exc()
+            self.send(sender, BenchmarkFailure(parsed_exception))
```

Comment:
Can we stop using the word `parsed_exception`? It's now a traceback. (I guess we don't need a variable at all.)